apps/server/src/modules/task/service/task-copy.service.ts

```diff
@@ -22,9 +23,10 @@ export class TaskCopyService {
 	) {}
 
 	async copyTask(params: TaskCopyParams): Promise<CopyStatus> {
-		const { originalTask, user, destinationCourse, destinationLesson } = params;
+		const { user, destinationLesson } = params;
+		const originalTask = await this.taskRepo.findById(params.originalTaskId);
 
-		const taskCopy = await this.copyTaskEntity(params, originalTask, user, destinationCourse, destinationLesson);
+		const taskCopy = await this.copyTaskEntity(params, originalTask, user, params.destinationCourse, destinationLesson);
```

Comment:
why we remove consts? I think it is better to keep those consts than use params.destinationCourse or params.originalTaskId

Reply:
At first, targetCourse was changed to targetCourseId as in the case of tasks, but after feedback from Thomas for BC-2339 I changed it back to targetCourse. In my rush I forgot to add this to the constants, it's already been corrected